static/homepage/index.html

```diff
@@ -133,9 +119,13 @@
             <h1 class="hero-header-text large">Share anything on the web.</h1>
             <h4 class="hero-tagline">Save, highlight, send
             </h4>
-            <p>
+            <p class="if-firefox">
```

Comment:
The JS adds a class to body, and the CSS uses selectors to hide or show these classes based on the body class.  I put a comment into the page JS to explain.